best-practices/largescale-deep-learning/Training/README.md

```diff
@@ -355,3 +355,7 @@ Nebula Checkpointing improves on standard model checkpointing by saving models 1
   |1|25B|16|	8|	1|	1|	119.42|	4.173	|69.7%|
   |2|20B|8|	8|	1	|1	|117.71	|2.51	|78.5%|
   |3|20B|8|	8|	1	|2|	123.52	|2.63	|80.1%|
+- ### **Pretraining a vision model**
+  Pretraining a vision model can be done in the supervised or the self-supervised learning paradigm. Similar considerations as for language models apply, i.e. pretraining produces good general features, it brings benefits in specialized domains with large amounts of unlabeled data, and it may be costly.
```

Comment:
can we merge this section under ViT Pretrain?

Reply:
That would make the distinction between language and vision models less visible. How about moving the paragraph under a "Pretraining models" subsection?